zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java

```diff
@@ -647,7 +650,8 @@ private void updateMessageState() {
 			if (incoming) {
 				String author = _messageIn.getAuthor().getDisplayName(); // Is message from channel?
 				if (author != null && author.length() > 0) {
-					_txtMessageName.setText(_messageIn.getFrom().getDisplayName() + " \\ " + author); // Show channel and author names
+					// Show channel and author names
+					_txtMessageName.setText(String.format(getResources().getString(R.string.message_in_from_channel), _messageIn.getFrom().getDisplayName(), author));
```

Comment:
You shouldn't need to call String.format() or getResources() here... it appears that the string resource already includes formatting specifiers so you should be able to call like this and let getString handling the formatting for you:

```
getString(R.string.message_in_from_channel, _messageIn.getFrom().getDisplayName(), author)
```